src/engine/BMGame.php

```diff
@@ -1882,13 +1882,13 @@ public function set_turbo_sizes(array $turboSizeArray) {
             $die = $this->playerArray[$playerIdx]->activeDieArray[$dieIdx];
 
             if (!$die->has_skill('Turbo')) {
-                $this->message = 'Cannot set turbo size for a non-turbo die';
-                return FALSE;
+                // ignore any turbo values supplied for non-turbo dice
```

Comment:
So, there are two changes here.

The second one, where the backend accepts and ignores turbo values from turbo dice that are not being changed, is necessary because of the UI change that leaves the selectors visible but disabled.

The first, where the backend accepts and ignores turbo values from non-turbo dice, is not necessary, but was done for consistency, so that this function consistently accepts irrelevant inputs and ignores them silently. If you want me to reverse this change, I can do so.